packages/devtools_app/lib/src/network/flutter/network_screen.dart

```diff
@@ -76,20 +85,23 @@ class NetworkScreenBodyState extends State<NetworkScreenBody> {
     return Row(
       children: [
         recordButton(
```

Comment:
if you instead made recordButton, pauseButton,  and clearButton widgets instead of methods, you could eliminate these keys.

Reply:
That might be a better change for a follow up CL since these methods are used in various other places.